modules/io/src/common/xviz-data.js

```diff
@@ -35,15 +35,17 @@ import {XVIZ_FORMAT} from './constants';
 // - arraybuffer which is a JSON string
 // - JSON object
 // - arraybuffer which is a GLB
+// opts.messageType is the XVIZ Envelope 'type', i.e. one of ('xviz/state_update', 'xviz/metadata', etc.)
```

Comment:
This is a bit confusing actually.  I think we want
`opts.messageType is the Message contained in data.  If supplied it assume data does not have an Envelope.`

Reply:
you mean `MessageType` no `Message` right?

Reply:
Yes.  so `messageType` is the optional argument that tells us the **type** of the **data**.   lots of similar words for slightly different semantic meaning here so I leave it to your judgement.  but i'd avoid mixing `messageType` and `MessageType`  together ;)